src/app/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import styles from "./Profile.module.css";
+import NavBar from "../../components/NavBar/NavBar";
+import ProfileCard from "../../components/ProfileCard/ProfileCard";
+import { UserDog, UserSitter } from "../../../types";
+import DogIcon from "../../components/Icons/DogIcon";
+
+function Profile(): JSX.Element {
+  const [user, setUser] = useState<UserSitter | UserDog | null>(null);
+
+  useEffect(() => {
+    fetch("/api/users/me")
+      .then((response) => response.json())
+      .then((user) => setUser(user));
+  }, []);
+
+  function calculateAge(birthYear: number): number {
+    const currentYear = new Date().getFullYear();
+    return currentYear - birthYear;
+  }
+
+  if (!user) {
+    return <p>No user found</p>;
+  }
+
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <NavBar />
+      </header>
+      <main className={styles.main}>
+        <ProfileCard
+          imgSrc={user.photo}
+          name={user.name}
+          mail={user.email}
+          phone={user.phone}
+          city={user.city}
+          info={user.type === "dog" ? "Alter: " : "Erfahrung: "}
+          info2={
+            user.type === "dog" ? calculateAge(user.birthYear) : user.experience
+          }
```

Comment:
This doesn't need to be done inline and can be done with to optional props like: `experience` and `age`